OJP/OJP_ModesSupport.xsd

```diff
@@ -78,16 +108,41 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
-	<xs:simpleType name="ContinuousModesEnumeration">
+	<xs:simpleType name="ContinuousModesOfOperationEnumeration">
 		<xs:annotation>
-			<xs:documentation>Types of cmodes that  run at any time without a timetable.</xs:documentation>
+			<xs:documentation>Types of MODES OF OPERATION that run at any time without a timetable and that are not classic flexible services.</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
 			<xs:enumeration value="walk"/>
-			<xs:enumeration value="demandResponsive"/>
-			<xs:enumeration value="replacementService"/>
```

Comment:
Where are those two going?

Reply:
demandResponsive is now in TimedMode. replacementService is gone. Was not on the list. I check.

Reply:
replacementService now also in TimedMode....

Reply:
If I remember correctly, we will have another discussion about demandResponsive being part of ContinuousModesOfOperationEnumeration,  or adding the GeoPosition to TimedLegs. I am in favor of adding it back to ContinuousModesOfOperationEnumeration.

Reply:
@herlitze  We have a problem there: Then area based demand responsive is a ContinuousLeg and not a TimedLeg. And I think it was also you who suggested that we should use a TimedLeg for every type of demand responsive public transport.  Can you clarify?

Reply:
@ue71603 I remember it that way that I agreed, but I was not the one who suggested removing demand responsive from ContinousLeg. We were not aware of the problems of mapping demand responsive transport serving addresses in a TimedLeg. With this UseCase we have to rethink the decision.

In the current state of the PullRequest we can pass a GeoPosition in addition to the existing StopPointGroup. If I recall our discussion correctly, if we want to keep demand responsive only in TimedLegs, we will need to support addresses as well, so the whole Place-structure is needed.
I dislike the idea, but the plan was for us to talk about the problem again, right?

Reply:
changed